conda.recipe/meta.yaml

```diff
@@ -11,11 +11,7 @@ build:
   # so we have to exempt them from the list of initial files
   # for conda-build to realize they should be included.
   always_include_files:
-    - bin/activate
-    - bin/deactivate
-    - bin/conda
-  entry_points:
-    - conda = conda.cli.main:main
+    - bin/conda  [unix]
```

Comment:
@groutr why not use `entry_points`? Against @ilanschnell's preference, I refactored `setup.py` to be all-in on setuptools.  So, `entry_points` there, with `conda build` overwriting the crappy `bin/` code setuptools creates.  We can also now `rm -rf bin/conda` in the repo I think.